python/lsst/ts/ess/data/ess_temperature_4ch_c.py

```diff
@@ -0,0 +1,34 @@
+# This file is part of ts_ess.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["EssTemperature4ChC"]
+
+
+class EssTemperature4ChC:
+    """Mock class that contains the 4 channel temperature data.
+    """
+
+    def __init__(self):
+        self.timestamp = None
+        self.TemperatureC01 = None
```

Comment:
temperatureC01 is lower case in XML, shall be lower case here as well

Reply:
That's correct. As I explained before, the current SALPY EssTemperature4ChC class has it with an upper case T and I cannot break that API. The fact that it should be with a lower case t will be addressed in the near future.